### DISCUSSION TITLE
[NEW] Server switcher

### DISCUSSION THREAD

**Discussion by KuroRaiken** [aXenDeveloper/ips-app-advanced-serverlist#51] [Ideas]
Add Server Switching Feature
![zczxc png 9cafcb3e3877e020f716081ab39892fc](https://user-images.githubusercontent.com/103626579/163715475-74f50bdc-eabc-404e-bd3f-e90fe50c4a20.png)

**Comment by aXenDeveloper** [comment]
Do you mean pagination?

**Comment by KuroRaiken** [comment]
> Do you mean pagination?

I don't speak English well, so I can't say exactly what I meant by a new feature in this app to be displayed on the site where I saw it and would like it to be added in the future.
Actually on this [site](https://cs-creativ.pl/).

**Comment by aXenDeveloper** [comment]
No, this feature will not be available.

**Comment by KuroRaiken** [comment]
> Нет, эта функция будет недоступна.

And you can find out why it will not be available?

**Comment by aXenDeveloper** [comment]
I don't want this in my app.

**Comment by KuroRaiken** [comment]
> Я не хочу этого в своем приложении.
I understood you.
Then we will do it ourselves.

**Comment by BlackYuzia** [comment]
@aXenDeveloper I don't sure. Maybe he mean not a switch or pages, this looks like `slide groups` or something. Example is [here](https://vuetifyjs.com/en/components/slide-groups/), but is for vue.
When you could sroll left or right and see other items. 

I think, that a second method for display servers (instead of table or two column table). Looks good. If you wnat, you could create, a poll. If a lot of users need this, we  ( I MEAN YOU :eyes: ) could make it. 

Note: On site, this widget have two display types. `slide groups` and table (check third image)

![image](https://user-images.githubusercontent.com/25010528/164993607-17c71476-d35f-43f6-9551-5bcaed7f3685.png)
![image](https://user-images.githubusercontent.com/25010528/164993717-ce92ab22-f96c-4cf1-a309-2e430e6a3b40.png)
![image](https://user-images.githubusercontent.com/25010528/164993718-a7a67859-52db-4879-aa38-9aa415a2ab80.png)

**Reply by KuroRaiken** [reply]
> 

Yes, that's what I was talking about)

**Reply by KuroRaiken** [reply]
> @aXenDeveloper I don't sure. Maybe he mean not a switch or pages, this looks like `slide groups` or something. Example is [here](https://vuetifyjs.com/en/components/slide-groups/), but is for vue. When you could sroll left or right and see other items.
> 
> I think, that a second method for display servers (instead of table or two column table). Looks good. If you wnat, you could create, a poll. If a lot of users need this, we ( I MEAN YOU 👀 ) could make it.
> 
> Note: On site, this widget have two display types. `slide groups` and table (check third image)
> 
> ![image](https://user-images.githubusercontent.com/25010528/164993607-17c71476-d35f-43f6-9551-5bcaed7f3685.png) ![image](https://user-images.githubusercontent.com/25010528/164993717-ce92ab22-f96c-4cf1-a309-2e430e6a3b40.png) ![image](https://user-images.githubusercontent.com/25010528/164993718-a7a67859-52db-4879-aa38-9aa415a2ab80.png)

Any ideas how to implement this in this application?

**Reply by BlackYuzia** [reply]
> Any ideas how to implement this in this application?

I think I know how it should be, but I work in a bit other category. So, I think, I can't do it right now :eyes:

<details>
  <summary>Translated to ru</summary>
  Иначе говоря, я знаю как это должно быть написано, но писать я это, конечно же, не буду :eyes: 
  Ну, одна из причин это в том что я не хочу разбирать тонны php кода вперемешку с html.
  К тому же, мне придется на пыхе писать. А я как бы и не по пыхе. :eyes:
    
  **Поищи мб пыхо кодера, может он за условную х сумму разберет код и напишет.**
  
</details>

**Comment by aXenDeveloper** [comment]
I don't want to implement this function.

The application is open source, so you can always help me with its development through pull requests.
https://github.com/aXenDeveloper/ips-app-advanced-serverlist/pulls

**Reply by KuroRaiken** [reply]
> Я не хочу реализовывать эту функцию.
> 
> Приложение с открытым исходным кодом, поэтому вы всегда можете помочь мне с его разработкой через пулл-реквесты. https://github.com/aXenDeveloper/ips-app-advanced-serverlist/pulls

It is suggested to add something similar to this function, which will be displayed when you click on a server, will show server information (for a large number of servers), when click show, it will show server information.
![123a](https://user-images.githubusercontent.com/103626579/165135138-6485bf8d-812a-46b1-81a7-286dd10cb33b.png)
